hangman/word_generator.py

```diff
@@ -0,0 +1,17 @@
+from requests import get
+import random
+from typing import List
+
+
+class WordGenerator:
+    def __init__(self, url) -> None:
+        self.api: str = url
+        self.words: List[str] = []
+
+    def get_data(self) -> None:
+        data: str = get(self.api).text
```

Comment:
also this is calling an API, have you considered what if this API fails, this generates a dependancy of your product on a third party service, if that goes down  your system stops working, but that might be unavoidable how do you handle these errors gracefully?